celery_prometheus_exporter.py

```diff
@@ -36,7 +40,10 @@
     'celery_workers', 'Number of alive workers')
 LATENCY = prometheus_client.Histogram(
     'celery_task_latency', 'Seconds between a task is received and started.')
-
+QUEUE_LENGTH = prometheus_client.Gauge(
+    'celery_queue_length', 'Number of tasks in a redis queue.',
```

Comment:
The metric should no longer refer to redis anymore. Does "backend queue" sound better here?

Reply:
For me, looks like ppl search for problems in celery queues, so  that 'celery_queue_length' hide an internal detail that is the celery backends. What do you think?

Reply:
sounds good 😄